include/picongpu/particles/Particles.kernel

```diff
@@ -211,15 +211,22 @@ namespace picongpu
             // relative offset (in cells) to the supercell (including the guard)
             DataSpace<simDim> const superCellOffset = block * SuperCellSize::toRT();
 
-            PMACC_SMEM(acc, mustShift, int);
+            /* worker status flag to mark that a particle is leaving the supercell
+             * 1 if at least one particle is leaving the supercell else 0
+             */
+            int mustShift = 0;
+            /* shared memory status flag that a particle is leaving the supercell
+             * 1 if at least one particle is leaving the supercell else 0
+             */
+            PMACC_SMEM(acc, sharedMustShift, int);
```

Comment:
Not important, but why not a fixed-sized type here?

Reply:
I would prefer to tell those comments in names of the variables, as now they describe how they operate and not what they do.
E.g.
`mustShift` -> `mustShiftCurrentParticle`
`sharedMustShift` -> `mustShiftSupercell`

Reply:
I will rename it next week.

Reply:
Maybe `mustShift` -> `hasLeavingParticle`